include/game/multiplayer/player_unk_1.h

```diff
@@ -4,7 +4,7 @@
 #define GUARD_MULTIPLAYER_PLAYER_UNK_1_H
 
 #include "global.h"
-#include "core.h"
+#include "core.h" // for MultiSioData
```

Comment:
At some point we move this out of core. It's unrelated

Reply:
Sounds good!